app/code/Bulbulatory/Recommendations/Model/Total/Quote/RecommendationDiscount.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+
+namespace Bulbulatory\Recommendations\Model\Total\Quote;
+
+
+use Bulbulatory\Recommendations\Helper\Config;
+use Bulbulatory\Recommendations\Helper\Recommendation as RecommendationHelper;
+use Magento\Customer\Model\Customer;
+use Magento\Quote\Api\Data\ShippingAssignmentInterface;
+use Magento\Quote\Model\Quote;
+use Magento\Quote\Model\Quote\Address\Total;
+use Magento\Quote\Model\Quote\Address\Total\AbstractTotal;
+use Magento\Customer\Model\Session;
+
+/**
+ * Class RecommendationDiscount
+ * @package Bulbulatory\Recommendations\Model\Total\Quote
+ */
+class RecommendationDiscount extends AbstractTotal
+{
+    /**
+     * @var RecommendationHelper
+     */
+    private $recommendationHelper;
+    /**
+     * @var Customer
+     */
+    private $customer;
+    /**
+     * @var Config
+     */
+    private $config;
+
+    /**
+     * RecommendationDiscount constructor.
+     * @param Session $session
+     * @param RecommendationHelper $recommendationHelper
+     * @param Config $config
+     */
+    public function __construct(Session $session, RecommendationHelper $recommendationHelper, Config $config)
+    {
+        $this->customer = $session->getCustomer();
+        $this->recommendationHelper = $recommendationHelper;
+        $this->config = $config;
+    }
+
+    /**
+     * @param Quote $quote
+     * @param ShippingAssignmentInterface $shippingAssignment
+     * @param Total $total
+     * @return $this|RecommendationDiscount
+     */
+    public function collect(
+        Quote $quote,
+        ShippingAssignmentInterface $shippingAssignment,
+        Total $total
+    )
+    {
+        parent::collect($quote, $shippingAssignment, $total);
+        if (!$this->config->isRecommendationsModuleEnabled()) return $this;
```

Comment:
Raczej trzymajmy się nawiasów, wtedy bardziej przejrzyście to wygląda.
```suggestion
        if (!$this->config->isRecommendationsModuleEnabled()) {
            return $this;
        }
```